builder/scripts.d/50-libopus.sh

```diff
@@ -31,9 +31,38 @@ ffbuild_dockerbuild() {
         return -1
     fi
 
+    if [[ $TARGET == linux* || $TARGET == mac* ]] && [[ $TARGET == *arm64 ]]; then
+        myconf+=(
+            --with-NE10-libraries="$FFBUILD_PREFIX"/lib
+            --with-NE10-includes="$FFBUILD_PREFIX"/include/libNE10
+        )
+    fi
+
+    # reset CLFAGS because libopus may give up optimization if current CFLAGS contains value
+    CFLAGS_BACKUP="$CFLAGS"
+
+    # For some reason libopus will not add optimization flag, we have to set it ourselves
+    if [[ $TARGET == mac* ]]; then
+        export CFLAGS="-O3"
+    else
+        export CFLAGS="-O3 -fPIC -DPIC"
+    fi
+
     ./configure "${myconf[@]}"
```

Comment:
It seems like optimizations being completely disabled only happens on macOS builds? I don't see the `!DEFINED(__OPTIMIZED__)` warning on crosstool-ng builds.

Opus have this in [their macOS CI](https://github.com/xiph/opus/blob/7db26934e4156597cb0586bb4d2e44dccdde1a59/.github/workflows/autotools.yml#L47):
```yml
      - name: Configure
        run: CFLAGS="-mavx -mfma -mavx2 -O2 -ffast-math" ./configure --enable-float-approx ${{ matrix.config.buildconfig }}
```

So maybe this will be enough for both platforms:
```sh
    # Override previously set -O(n) option and the CC's default optimization options.
    CFLAGS="$CFLAGS -O3" ./configure "${myconf[@]}"
```